packages/crud/src/crud/useRouterQueryFilters.ts

```diff
@@ -48,13 +48,14 @@ const useRouterQueryFilters = (args) => {
 
         // Set relationalObjectValue to render the correct title
         const { pk = 'title' } = filterField.module
+        const { optionLabelKey = 'title' } = filterField
         const relationalObjectValue =
           typeof relationalObject === 'object'
-            ? relationalObject[pk]
+            ? relationalObject[optionLabelKey] ?? relationalObject[pk]
```

Comment:
@clodal this allows downstream apps to provide a custom label key e.g for assignee_id, the label key should be `full_name` and not `title`

Reply:
Thanks how is this different from pk tho?

Reply:
Should we do filterField.pk instead of filterField.optionLabelKey?

Reply:
@clodal For `assignee` filter, we want to display the `full_name` instead of `title`, and for this field: optionLabelKey = full_name, pk = title. So for some model fields, downstream apps might want to customize which field to be display as label